daphne/src/testing.rs

```diff
@@ -136,6 +136,27 @@ impl MockAggregator {
         }
     }
 
```

Comment:
`check_report()` is not a very descriptive name. Would you mind adding a comment describing what this function does?

Reply:
I've added the comment.